src/TableOfContents/index.tsx

```diff
@@ -198,7 +246,7 @@ export function TableOfContents<T extends TableOfContentsItem = TableOfContentsI
               {title}
             </Button>
           </div>
-          <div className="h-full flex justify-end">{comp}</div>
+          <div className="h-full flex flex-col justify-end">{comp}</div>
```

Comment:
Hmm - how did this work without `flex-col` before?

Reply:
Hmm. Because there was just one component inside it? And this is for mobile.